lib/metrics.go

```diff
@@ -157,6 +163,7 @@ func (l LatencyMetrics) Quantile(nth float64) time.Duration {
 
 func (l *LatencyMetrics) init() {
 	if l.estimator == nil {
+		l.Min = time.Duration(math.MaxInt64)
```

Comment:
What makes this line necessary?

Reply:
Because `Int` value is initialize with `0`  .
But, I forgot measures were already taken at [Line 152].(https://github.com/tsenart/vegeta/pull/446/files/3fa39a80ed8dc214606ea630304620fb65a84db2#diff-d60ed9ab81a6041797ea9d191d7bb753R152)

I remove it. thank you for check.